TestingApp/Settings/SettingsViewController.swift

```diff
@@ -38,21 +38,31 @@ class SettingsViewController: UIViewController {
         cells: []
     )
 
-    var theme: Theme = Theme()
-    var conf: Configuration { loadConf() }
-    var queueID: String { loadQueueID() }
-    var visitorContextAssetId: String { loadVisitorContextAssetId() }
-    var features: Features { loadFeatures() }
+    private var props: Props
+
+    init(
```

Comment:
NIT: I don't think that having the `props` in a new line yield any benefits, as it's actually less readable than having it all together on the same line.

Reply:
fixed - 1dd9a6d247757d77b9f69974e2e98c719d91741e